src/operations/get_more.ts

```diff
@@ -56,7 +62,40 @@ export class GetMoreOperation extends AbstractOperation {
         new MongoRuntimeError('Getmore must run on the same server operation began on')
       );
     }
-    server.getMore(this.ns, this.cursorId, this.options, callback);
+
+    const collection = this.ns.collection;
```

Comment:
I was thinking of testing the logic between lines 66 and 96.  killCursors is simple enough We don't explicitly test it anywhere that I know of but unit testing the actual structure of the commands that we create would be valuable in letting us know exactly what options are / aren't supported for each command and how we use them

Reply:
Did you see: https://github.com/mongodb/node-mongodb-native/pull/3317/files#diff-618e4bf208953ae05a6294c96859d7287b6c6b0e39dd3a827f581cce73f3f196R13? Or are you just pointing out that was what you meant would be good coverage

Reply:
I mistyped in my previous comment.  I think we adequately cover killCursors.  But I don't think we cover the command construction of `getMore` sufficiently, and it's true we didn't have coverage for that before but I think we should add it since you're modifying it!

Reply:
Added some basic assertions, we can add more with the maxTimeMS bug fix work

Reply:
looks good, thanks